p/themes/base-theme/template.rtl.css

```diff
@@ -1418,13 +1419,25 @@ br {
 }
 
 #close-slider.active {
+	background: rgba(0, 0, 0, 0.2);
+	font-size: 0;
```

Comment:
I'm not sure what the intent here is. Presumably one of display: none, visibility: hidden or visibility: collapse would communicate the intent properly. ^_^

Reply:
The reason for `font-size: 0`:

`<a href="#" id="close-slider"<?= $class ?>>❌</a>`

Similar to: 
https://github.com/FreshRSS/FreshRSS/blob/edge/app/layout/aside_feed.phtml#L118

https://github.com/FreshRSS/FreshRSS/blob/edge/p/themes/base-theme/template.css#L1763-L1774